certbot-ci/certbot_integration_tests/nginx_tests/test_main.py

```diff
@@ -1,14 +1,14 @@
 """Module executing integration tests against certbot with nginx plugin."""
-import os
+import os  # pylint: disable=os-module-forbidden
 import ssl
 
 import pytest
 
 from certbot_integration_tests.nginx_tests import context as nginx_context
 
 
-@pytest.fixture()
-def context(request):
+@pytest.fixture(name='context')
+def test_context(request):
```

Comment:
See https://github.com/certbot/certbot/pull/8450/files#r522379272

Reply:
Masking context from outer scope. I could silence it, but I think this is more correct.

Reply:
I totally agree.